packages/wrangler/src/config/config.ts

```diff
@@ -183,6 +185,18 @@ export interface ConfigFields<Dev extends RawDevConfig> {
 	keep_vars?: boolean;
 }
 
+// Pages-specific configuration fields
```

Comment:
keeping this separate so that when things converge it'll be easy to remove these code paths :)